tests/test_clicks.py

```diff
@@ -0,0 +1,115 @@
+from poprox_storage.repositories.clicks import DbClicksRepository
+from poprox_concepts import Account
+from datetime import datetime
+
+import json
+import pytest
+import os
+from sqlalchemy import create_engine, text
+from uuid import uuid4
+
+db_password = os.environ.get("POPROX_DB_PASSWORD", "")
+db_port = os.environ.get("POPROX_DB_PORT", "")
+
+DEFAULT_PG_URL = f"postgresql://postgres:{db_password}@127.0.0.1:{db_port}/poprox"
+
+
+@pytest.fixture(scope="session")
+def pg_url():
+    """
+    Provides base PostgreSQL URL for creating temporary databases.
+    """
+    return os.getenv("CI_POPROX_PG_URL", DEFAULT_PG_URL)
+
+
+def test_get_active_survey(pg_url: str):
+    engine = create_engine(pg_url)
+    with engine.connect() as conn:
+        conn.execute(text("delete from clicks;"))
+        conn.execute(text("delete from newsletters;"))
+        conn.execute(text("delete from articles;"))
+
+        dummy_click_1 = uuid4()
+        dummy_click_2 = uuid4()
+        dummy_user_1 = uuid4()
+        dummy_user_2 = uuid4()
+        dummy_newsletter = uuid4()
+
+        dummy_article_1 = uuid4()
+        dummy_article_2 = uuid4()
+
+        query_0 = text(
+                f'''insert into accounts(account_id, email, status, source) values (:account_id, 'user-1@gmail.com', '', '');'''
+            )
+        query_1 = text(
+                f'''insert into clicks(click_id, account_id, newsletter_id, article_id, created_at)
+                    values (:click_id_1, :account_id, :newsletter_id, :article_id_1, '2024-06-12 09:55:22');'''
+            )
+        query_2 = text(
+                f'''insert into clicks(click_id, account_id, newsletter_id, article_id, created_at)
+                    values (:click_id_2, :account_id, :newsletter_id, :article_id_2,  '2024-07-14 12:55:22');'''
+            )
```

Comment:
`query_1` and `query_2` look like `DbClicksRepository.track_click_in_database()`. I see that this is setting the `created_at` field, which that method doesn't yet provide a way to do. Could you add an optional `created_at` param to that method and then use it here?

Reply:
Ahh, found the same complaint about double db connection here as well 😅